test/modules/appeals-service/AppealsService.test.ts

```diff
@@ -66,11 +67,11 @@ describe('AppealsService', () => {
                         },
                     }
                 )
-                .reply(201, {}, { 'location': RESOURCE_LOCATION });
+                .reply(201, appealId, { 'location': RESOURCE_LOCATION });
 
             await appealsService.save(appeal, BEARER_TOKEN)
-                .then((response) => {
-                    expect(response).to.equal(RESOURCE_LOCATION);
+                .then((response: string) => {
+                    expect(response).to.equal(appealId);
```

Comment:
we can still expect the response to contain the location header?

Reply:
the response is now the appealId not the header

Reply:
the response which is actually just the result of resolving the promise